backend/routes.py

```diff
@@ -0,0 +1,145 @@
+import uuid as u
+import copy
+import backend.pike as pike
+import tests.api_fixtures as api_mocks
+
+
+## Read calls
+# c.iii.1 (?) get available agents
+@pike.api.get("/agents")
+def get_public_agents() -> list[dict]:
+    """
+    Get a list of all public agent types.
+    """
+    return [api_mocks.mock_agent_one_skill, api_mocks.mock_agent_two_skills]
+
+
+# 1.d.  User settings
+@pike.api.get("/user/{user_id}")
+def get_user_info(user_id: str) -> dict:
+    """
+    Provides full info about a user given their user_id.
+    """
+    return api_mocks.mock_user
+
+
+# 1.a.  get my agents
+@pike.api.get("/user/{user_id}/agents")
+def get_user_agents(user_id: u.UUID) -> list[dict]:
+    """
+    Get all agents the user has chosen to enable.
+    """
+    return api_mocks.mock_user["agents"]
+
+
+# 1.b.  get compact chat representations
+@pike.api.get("/user/{user_id}/chats")
+def get_user_chats_compact(user_id: str) -> list[dict]:
```

Comment:
why *_compact? if we call the one with messages as *_history, you can drop compact.